src/Sfa.Das.ApprenticeshipInfoService.Api/Controllers/SearchController.cs

```diff
@@ -16,14 +16,21 @@ namespace Sfa.Das.ApprenticeshipInfoService.Api.Controllers
 
     public class SearchController : ApiController
```

Comment:
Would it be better to us a library such as https://github.com/Microsoft/aspnet-api-versioning to allow us to easily manage the versioning of the API going forward?

Reply:
Have implemented the use of the library mentioned. There's a new V3 version that our new endpoint will sit under.